Library/Homebrew/cmd/update-report.rb

```diff
@@ -671,7 +671,7 @@ def dump(updated_formula_report: true)
     return if outdated_formulae.blank? && outdated_casks.blank?
 
     outdated_formulae = outdated_formulae.count
-    outdated_casks = outdated_casks.count
+    outdated_casks = T.must(outdated_casks).count
```

Comment:
Can we initialize these with `[]` instead to avoid this?

Reply:
Yep! One sec.